data/remnant/remnant missions.txt

```diff
@@ -1060,6 +1060,326 @@ mission "Remnant: Learn Sign Follow Up"
 			`	"Congratulations, Captain <last>!" the director signs. "Your fluency is good enough that even those unwilling to sing with a stranger should be comfortable working with you. At this rate you will be one of us in more ways than just by blood. I look forward to seeing what results from your work with us." The director is interrupted by the arrival of other Remnant asking for something and bids you farewell with an apologetic gesture.`
 				decline
 
+	
+mission "Remnant Rescue"
+	job
+	repeat
+	name "Search and Rescue: Scout"
+	description "A ship, the <npc>, has gone missing during a scouting mission in the Ember Wastes. Find them and help them return to <destination>."
+	source
+		government "Remnant"
+	to offer
+		has "Remnant: Learn Sign Follow Up: offered"
+		random < 30
+	npc accompany save
+		system
+			attributes "ember waste"
+			not attributes "ka'het"
+			not attributes "inaccessible"
+			not attributes "graveyard"
+			not government "Remnant"
+			not system "Perfica"
+			not system "Farinus"
+		personality derelict timid plunders
+		government "Remnant"
+		fleet
+			names "remnant"
+			variant
+				Starling
+			variant
+				Gull
+	on complete
+		"rescue missions" ++
+		payment 200000
+		dialog
+			`Your communicator notifies you of a resource allocation of <payment>.`
+	on visit
+		dialog
+			`You have returned, yet the <npc> is still missing.`
+
+	
+ship "Rescue Dummy"
+	sprite "ship/behemoth"
+	attributes
+		"heat dissipation" 1
+		"hull" 1
+		"mass" 10
+		"drag" 1
+		"automaton" 1
+		"energy generation" 1
+		"thrust" 30
+		"turn" 30
+	outfits
+		"Suicide Gun"
+	gun 0 0
+
+outfit "Suicide Gun"
+	"category" guns
+	thumbnail "outfit/unknown"
+	"gun ports" -1
+	weapon
+		velocity 100
+		lifetime 10
+		reload 1
+		"blast radius" 900
+		"shield damage" 0.01
+		"hull damage" 1
+
+						
+mission "Remnant Rescue 1B"
+	job
+	repeat
+	name "Search and Rescue: Scout"
+	description "A ship, the <npc>, has gone missing during a scouting mission in the Ember Wastes. Find them and help them return to <destination>."
+	source
+		government "Remnant"
+	to offer
+		has "Remnant: Learn Sign Follow Up: offered"
+		random < 5
+	npc kill
+		system
+			attributes "ember waste"
+			not attributes "ka'het"
+			not attributes "inaccessible"
+			not attributes "graveyard"
+			not government "Remnant"
+			not system "Perfica"
+			not system "Farinus"
+		personality staying heroic nemesis unconstrained
+		government "Test Dummy"
+		fleet
+			names "remnant"
+			variant
+				"Rescue Dummy"
+		dialog
+			`As you enter the system, your sensors pick up a lot of debris. Further analysis confirms your worst fears. The <npc> is destroyed. You frantically spend the next few hours searching for survivors, your hope sinking with every minute that passes.`
```

Comment:
```suggestion
			`As you enter the system, your sensors pick up a lot of debris. Further analysis confirms your worst fears. The <npc> has been destroyed. You frantically spend the next few hours searching for survivors, your hope sinking with every passing minute.`
```

Reply:
Admittedly, something just doesn't feel right with the last sentence, but I'm not touching that.

Reply:
I'll commit your suggestion because it is better, but I'll definitely put some thought into improving it.